### PR TITLE
ENT-11371 MPF.md: Fixed many inconsistencies

### PR DESCRIPTION
**Pull request by olehermanse**
This file is shown in our docs and could use a cleanup.

Went over this file trying to fix as many obvious / easy inconsistencies as I could:
* Removed the use of h1, h5 and h6 headings.
* Added JSON highlighting to code blocks.
* Fixed invalid JSON (trailing commas, missing colons).
* Fixed inconsistent indentation / formatting of code blocks.
* Fixed capitalization of several headers to use sentence case.
* Fixed capitalization of CFEngine, HTTP, HTTPS, etc. in a few places.
* Added monospace font for binaries; cf-serverd, etc.
* Fixed inconsistent markdown syntax (especially what should be bold and which symbols to use for bulleted lists).
* Fixed small typos / grammer mistakes.
* Renamed some headings to make them easier to understand.

Ticket: ENT-11371

### CODE REVIEW

**Review by larsewi — APPROVED** [review]
💩

File: MPF.md [hunk 1]
```diff
@@ -19,26 +19,26 @@ various aspects of the system.
...
 
-Note: If you need to make modification to a shipped file consider opening a pull
+**Note:** If you need to make modification to a shipped file consider opening a pull
 request to expose the tunable into the ```def``` bundle.
 
 **Note:** `controls/def.cf` contains the defaults and settings for `promises.cf`
 and `controls/update_def.cf` contains the defaults and settings for `update.cf`.
 
-Since 3.7.8, 3.10.4, and 3.12.0 The class `cf_runagent_initated` is defined by
+Since 3.7.8, 3.10.4, and 3.12.0 The class `cf_runagent_initiated` is defined by
```
Comment (nickanderson) [inline MPF.md:36]:
```suggestion
**History:**

* In 3.7.8, 3.10.4, and 3.12.0 the class `cf_runagent_initiated` is defined by
```

File: MPF.md [hunk 2]
```diff
@@ -1479,31 +1473,32 @@ be defined in an [augments file][Augments].
...
-By default cf-serverd holds an open connection for client initiated for 30
+By default `cf-serverd` holds an open connection for client initiated for 30
 seconds. In some environments this value may need to be increased in order for
 report collection to finish. Once the connection has been open for longer than
 the specified seconds it is closed.
 
 The window of time can be controled by setting `def.control_server_collect_window`.
 
-For example, enable client initated reporting for all hosts with a 10 minute interval and hold the connection open for 90 seconds.
+For example, enable client initiated reporting for all hosts with a 10 minute
```
Comment (craigcomstock) [inline MPF.md:1496]: why change one line (nice) to two lines here? Doesn't help result right?

Comment (craigcomstock) [reply MPF.md:1496]: though I do see just below and above we are 80-column formatting in general :( so I guess, OK. 😢

File: MPF.md [hunk 3]
```diff
@@ -1623,10 +1617,9 @@ memory related probes on policy servers:
 
 ### Configure Enterprise Mission Portal Docroot
 
-Primarily for developer conveniance, this setting allows you to easily disable the enforcement that the webapp consists of the packaged files in the docroot used for Mission Portal.
-
+Primarily for developer convenience, this setting allows you to easily disable the enforcement that the webapp consists of the packaged files in the docroot used for Mission Portal.
```
Comment (craigcomstock) [inline MPF.md:1620]: make 80-column?

File: MPF.md (hunk 2 above)
Comment (olehermanse) [reply MPF.md:1496]: Just trying to increase consistency, i.e. be consistent with what's around.

**Review by craigcomstock — COMMENTED** [review]
good changes generally.

I would prefer we did one line per sentence style here but I guess some folks will read this in a console so 80-ish columns makes some sense.

thanks for all this busy work! appreciate it!

**Comment by nickanderson** [comment]
I opened a PR in docs to see the result of this: https://github.com/cfengine/documentation/pull/3221

**Review by craigcomstock — APPROVED** [review]

**Comment by nickanderson** [comment]
> good changes generally.
> 
> I would prefer we did one line per sentence style here but I guess some folks will read this in a console so 80-ish columns makes some sense.
> 
> thanks for all this busy work! appreciate it!

humm, maybe we add the necessary doc comments to enable line wrapping in the editors. I sort of prefer 80 column limit but do find myself using long lines with visual wrapping more and more.

**Comment by olehermanse** [comment]
> I would prefer we did one line per sentence style here but I guess some folks will read this in a console so 80-ish columns makes some sense.

@craigcomstock I would also prefer it, but it didn't seem worth it, big file to edit.

File: MPF.md [hunk 4]
```diff
@@ -580,30 +578,29 @@ component.
...
 {
   "classes": {
     "clear_persistent_disable_cf_monitord": [ "redhat" ]
   }
 }
 ```
 
-##### Using augments
-##### agents_to_be_disabled
+#### Augments variable: agents_to_be_disabled
```
Comment (nickanderson) [inline MPF.md:589]:
```suggestion
#### Variable: def.agents_to_be_disabled
```

**Review by nickanderson — APPROVED** [review]
Just a couple suggestions.

File: MPF.md [hunk 5]
```diff
@@ -548,10 +549,7 @@ When enabled, the policy will render systemd unit files in `/etc/systemd/system`
 
 When enabled, the policy will make sure that all units are enabled, unless they have been disabled by a persistent class or are explicitly listed as an agent to be disabled.
 
-#### Enable or disable CFEngine components
-
-##### Using persistent classes
-###### persistent\_disable\_*DAEMON*
+#### Class: persistent\_disable\_*DAEMON*
```
Comment (nickanderson) [inline MPF.md:552]:
```suggestion
#### Class: default:persistent\_disable\_*DAEMON*
```

**Comment by nickanderson** [comment]
So fast docbuild won't pick up these changes until they are found in the nightly package.

**Pull request merged**